taipy_theme/assets/stylesheets/avaiga.css

```diff
@@ -1019,7 +1018,7 @@ div.tp-nav-button-group {
     display: flex;
     gap: 0.1rem;
     margin: 5px 0 5px 0;
-    width: calc(100% - .1rem);
+    width: calc(100% - 3rem);
```

Comment:
This must match the spacing between the two horizontal buttons in the nav section.
Are you really sure??